src/config/model.rs

```diff
@@ -47,7 +47,7 @@ impl Account {
 
     pub fn imap_passwd(&self) -> Result<String> {
         let passwd = run_cmd(&self.imap_passwd_cmd).chain_err(|| "Cannot run IMAP passwd cmd")?;
-        let passwd = passwd.trim_end_matches("\n").to_owned();
+        let passwd = passwd.trim_end().to_owned();
```

Comment:
With `trim_end` you also remove spaces, which are *in theory* valid chars for a password. I admit that *in practice* nobody uses this char in his password, but still. What do you think about:

```rust
let passwd = passwd
    .trim_end_matches(|c| c == '\r' || c == '\n')
    .to_owned();
```

Reply:
Make sense, I will update the patch.